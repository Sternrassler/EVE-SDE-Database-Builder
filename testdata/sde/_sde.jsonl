{"version":"20250101","lastModified":"2025-01-01T00:00:00Z","description":"EVE SDE Test Data"}
