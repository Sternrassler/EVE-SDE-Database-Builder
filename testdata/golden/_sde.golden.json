[
  {
    "version": "20250101",
    "releaseDate": null
  }
]